src/App.js

```diff
@@ -129,6 +158,27 @@ const App = () => {
                 exact
               />
               <Route path="/CitizenService/AllForums" element={<AllForums />} />
+
+              <Route
```

Comment:
You have added all of these appointment routes under the isForumsEnabledFlag. Add another flag for appointment booking and set these routes if that flag is enabled

Reply:
Done